greendots_plugin/plugin.py

```diff
@@ -317,7 +317,7 @@ def pytest_sessionstart(self, session):
         if not self.is_worker(session):
             return
 
-        self._status_file = StatusFile(status_file=open(os.path.join(self._log_path, f"status.{self.get_worker_id(session)}.jsonl"), "w"))
+        self._status_file = StatusFile(status_file=open(os.path.join(self._log_path, f"status.{self.get_worker_id(session)}.jsonl"), "a+"))
```

Comment:
This is invalid btw, appends are not guaranteed to be atomic in linux, so multiple processes writing to the same file (even in append mode) might interleave writes.